pkg/ctrl/server/schema.resolvers.go

```diff
@@ -175,6 +174,10 @@ func (r *hTTPChaosResolver) Annotations(ctx context.Context, obj *v1alpha1.HTTPC
 	return annotations, nil
 }
 
+func (r *hTTPChaosResolver) ClusterName(ctx context.Context, obj *v1alpha1.HTTPChaos) (string, error) {
```

Comment:
It is strange here. I guess it is because the kube go mod we relied on changed. 
You may need to turn the type of HTTPChaos.ClusterName to string here.

Reply:
I have no idea about what `ClusterName` is 🤔. It has been here 13 months ago.

Reply:
also cc @Hexilee

Reply:
`metadata.clusterName` is deprecated in 1.24 and removed in 1.25, since it is event not used, I would remove it from the graphql.

ref:

- https://github.com/kubernetes/kubernetes/pull/108717
- https://github.com/kubernetes/kubernetes/pull/109602